benchmark/finite_strain/pure_shear_tracer.dat

```diff
@@ -0,0 +1,5 @@
+# This is testdata for the Ascii file tracer generator
+# The last line is outside of the test domain and tests
+# that plugins simply ignore points outside of the domain
+# Columns: x_coordinate y_coordinate
+0.0 0.0
```

Comment:
The comment is puzzling since there is only one line, so to talk about the "last line" requires deeper thought than I can offer right now :-)